sysbuild: Fix nRF5340 network core updates (#17108)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

  .. note::
     This has security implications and may allow secrets to be leaked to the non-secure application in RAM.

> * Added the ``SB_CONFIG_MCUBOOT_NRF53_MULTI_IMAGE_UPDATE`` sysbuild Kconfig option to allow updating the network core on nRF5340 from external flash.

Reviewer: ```suggestion
* Added the ``SB_CONFIG_MCUBOOT_NRF53_MULTI_IMAGE_UPDATE`` sysbuild Kconfig option that enables updating the network core on the nRF5340 SoC from external flash.
```

Author: Fixed
